### ISSUE TITLE
Quantum Storage Unit: Annoying issue

### ISSUE THREAD

**Issue by L3thal1ty** [gigabit101/QuantumStorage#162]
So this is my first time using both RS (I greatly prefer AE2) and Quantum Storage, and I was doing a drive swap with the disk manipulator. When I pulled my main drive (64k) and was trying to push it all to a new 13486k drive, I noticed that the 64k was counting down, but nothing was appearing in my 13486k. Turns out, all these "missing" items were spilling out into my quantum storage units. The quantum storage unit has that one internal space for manually inputting items, and this is where the RS system was storing all my transferred stuff, despite having a disk in the drive. Granted, breaking and replacing the disk drive after disconnecting my quantum storages does seem to reset the priorities, but it leads up to a serious annoyance where I have to go clean out 100+ quantum storage units from the superfluous items.

TLDR: Quantum storage units don't always play nice with other mods. It would be super cool if we had a way to lock the unit so that it didn't accept any incoming items that did not match the item stored... you know, like a DSU does. 

Disclaimer: I am not super active on github, so if I didn't follow some exact issue reporting protocol, please forgive my ignorance.

**Comment by L3thal1ty** [comment]
I just noticed issue 161 and the upcoming lock for the QSU. Sorry for the inconvenience.
